app/test/process_proposal_test.go

```diff
@@ -165,7 +165,7 @@ func TestMessageInclusionCheck(t *testing.T) {
 // 		require.NoError(t, err)
 
 // 		require.NoError(t, err)
-// 		eds, err := da.ExtendShares(input.BlockData.OriginalSquareSize, shares)
+// 		eds, err := da.ExtendShares(input.BlockData.SquareSize, shares)
```

Comment:
I think we should be deleting these tests rather than leaving them commented out. Created https://github.com/celestiaorg/celestia-app/issues/1026